src/pages/FavoritePhrases.js

```diff
@@ -71,3 +75,24 @@ const PhrasesList = styled.ul`
 const PhraseItem = styled.li`
   width: 100%;
 `;
+
+const AddButtonLink = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+  padding: 0.5rem;
+  gap: 10px;
+  color: #19337a;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  min-height: 250px;
+  min-width: 300px;
+  border: 1px dashed #19337a;
+  background-color: #f9f9f9;
+  border-radius: 10px;
+`;
```

Comment:
This component looks very similar to the AllPhrases component, maybe you can combine them somehow to a single more general component. This might save you a lot of work when you want to establish a consistent styling.